### PR TITLE
enh(setup) add full mariadb config

### PR DESCRIPTION
**Pull request by Sims24**
## Description

The goal of this PR is to get rid of this stale PR: https://github.com/centreon/centreon-documentation/pull/226 . 

It provides more details on how to setup an optimized MariaDB configuration. 

## Target serie

- [X] 20.04.x
- [X] 20.10.x (master)

### CODE REVIEW

File: en/installation/installation-of-a-central-server/using-packages.md [hunk 1]
```diff
@@ -154,6 +154,28 @@ DROP USER '<USER>'@'<IP>';
...
+> read_buffer_size = 512K
+> read_rnd_buffer_size = 256K
+> max_allowed_packet = 128M
+> ```
+> 
+> Optionnaly, tune the memory and buffer utilization of the InnoDB engine powered 
+> tables. The example below applies to a database server with 8Gb RAM
+>  
+> ```shell
+> innodb_additional_mem_pool_size=1G
```
Comment (cgagnaire) [inline en/installation/installation-of-a-central-server/using-packages.md:176]:
```suggestion
```

Comment (cgagnaire) [reply en/installation/installation-of-a-central-server/using-packages.md:176]: innodb_additional_mem_pool_size is deprecated.

File: fr/installation/installation-of-a-central-server/using-packages.md [hunk 2]
```diff
@@ -160,6 +160,28 @@ DROP USER '<USER>'@'<IP>';
...
+> read_rnd_buffer_size = 256K
+> max_allowed_packet = 128M
+> ```
+> 
+> Il est également possible d'améliorer les performances des tables utilisant 
+> le moteur de stockage InnoDB avec les directives ci-dessous. Les valeurs 
+> ci-dessous sont applicables à une VM ayant au moins 8Go de RAM.
+>  
+> ```shell
+> innodb_additional_mem_pool_size=1G
```
Comment (cgagnaire) [inline fr/installation/installation-of-a-central-server/using-packages.md:182]:
```suggestion
```

Comment (cgagnaire) [reply fr/installation/installation-of-a-central-server/using-packages.md:182]: innodb_additional_mem_pool_size is deprecated.

File: fr/installation/installation-of-a-central-server/using-packages.md [hunk 3]
```diff
@@ -160,6 +160,28 @@ DROP USER '<USER>'@'<IP>';
 > innodb_file_per_table=1
 > open_files_limit=32000
 > ```
+> En sus des directives précédentes, il est fortement recommandé d'appliquer la
```
Comment (cgagnaire) [inline fr/installation/installation-of-a-central-server/using-packages.md:163]:
```suggestion

> En sus des directives précédentes, il est fortement recommandé d'appliquer la 
```

File: en/installation/installation-of-a-central-server/using-packages.md [hunk 4]
```diff
@@ -154,6 +154,28 @@ DROP USER '<USER>'@'<IP>';
 > innodb_file_per_table=1
 > open_files_limit=32000
 > ```
+>  
+> In addition to the directives above, it's strongly recommended to tune the
```
Comment (cgagnaire) [inline en/installation/installation-of-a-central-server/using-packages.md:158]:
```suggestion

> In addition to the directives above, it's strongly recommended to tune the 
```

Comment (cgagnaire) [reply en/installation/installation-of-a-central-server/using-packages.md:158]: To separate blocs.

File: fr/installation/installation-of-a-central-server/using-packages.md (hunk 3 above)
Comment (cgagnaire) [reply fr/installation/installation-of-a-central-server/using-packages.md:163]: To separate blocs.

**Review by cgagnaire — CHANGES_REQUESTED** [review]

File: en/installation/installation-of-a-central-server/using-packages.md (hunk 1 above)
Comment (Sims24) [reply en/installation/installation-of-a-central-server/using-packages.md]: Done

File: fr/installation/installation-of-a-central-server/using-packages.md (hunk 2 above)
Comment (Sims24) [reply fr/installation/installation-of-a-central-server/using-packages.md]: Done

**Review by cgagnaire — APPROVED** [review]

**Pull request merged**